assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -85,34 +85,39 @@ che.workspace.maven_options=-XX:MaxRAM=150m -XX:MaxRAMFraction=2 -XX:+UseParalle
 che.workspace.maven_server_java_options=-XX:MaxRAM=128m -XX:MaxRAMFraction=1 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10 -XX:MaxHeapFreeRatio=20 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -Dsun.zip.disableMemoryMapping=true -Xms20m -Djava.security.egd=file:/dev/./urandom
 
 # RAM limit default for each machine that has no RAM settings in environment.
+# To disable setting limit, value of -1 can be used.
 che.workspace.default_memory_limit_mb=1024
```

Comment:
this is che6 concept - right?

Reply:
Nope. That's everything which is not chePlugin/cheEditor